{
  "uid" : "70968d80fde21448",
  "name" : "test_buy_empty_basket",
  "fullName" : "test.test_main_page.TestMainPage#test_buy_empty_basket",
  "historyId" : "d6b5786680b38868ad821954821da58b",
  "time" : {
    "start" : 1650553748973,
    "stop" : 1650553748973,
    "duration" : 0
  },
  "description" : "\n        Попытка купить, если корзина пуста.\n        ",
  "descriptionHtml" : "<pre><code>    Попытка купить, если корзина пуста.\n</code></pre>\n",
  "status" : "skipped",
  "statusMessage" : "Skipped: missing notification",
  "statusTrace" : "('/home/runner/work/Test_with_a_team/Test_with_a_team/test/test_main_page.py', 22, 'Skipped: missing notification')",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "app",
    "time" : {
      "start" : 1650553749304,
      "stop" : 1650553749997,
      "duration" : 693
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x563223ff8ad3 <unknown>\n#1 0x563223d58568 <unknown>\n#2 0x563223d7bb37 <unknown>\n#3 0x563223d7723a <unknown>\n#4 0x563223db22ba <unknown>\n#5 0x563223dac4e3 <unknown>\n#6 0x563223d81d1a <unknown>\n#7 0x563223d82e75 <unknown>\n#8 0x563224026efd <unknown>\n#9 0x56322404019b <unknown>\n#10 0x563224028c65 <unknown>\n#11 0x563224040ec8 <unknown>\n#12 0x56322401c360 <unknown>\n#13 0x56322405ca68 <unknown>\n#14 0x56322405cbe8 <unknown>\n#15 0x5632240767fd <unknown>\n#16 0x7f14f1b2a609 <unknown>\n\n",
    "statusTrace" : "  File \"/home/runner/.local/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/Test_with_a_team/Test_with_a_team/conftest.py\", line 38, in app\n    driver = webdriver.Chrome(ChromeDriverManager().install(),\n  File \"/home/runner/.local/lib/python3.8/site-packages/selenium/webdriver/chrome/webdriver.py\", line 70, in __init__\n    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/home/runner/.local/lib/python3.8/site-packages/selenium/webdriver/chromium/webdriver.py\", line 93, in __init__\n    RemoteWebDriver.__init__(\n  File \"/home/runner/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 269, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/runner/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 360, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/home/runner/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 425, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 0
  }, {
    "name" : "register_user",
    "time" : {
      "start" : 1650553748976,
      "stop" : 1650553749303,
      "duration" : 327
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1650553735045,
      "stop" : 1650553735057,
      "duration" : 12
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.skip(reason='missing notification')"
  }, {
    "name" : "parentSuite",
    "value" : "test"
  }, {
    "name" : "suite",
    "value" : "test_main_page"
  }, {
    "name" : "subSuite",
    "value" : "TestMainPage"
  }, {
    "name" : "host",
    "value" : "fv-az266-753"
  }, {
    "name" : "thread",
    "value" : "2280-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_main_page"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 3,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "19cb6f767c638d8e",
        "reportUrl" : "https://ArkadiyVoronov.github.io/Test_with_a_team/6//#testresult/19cb6f767c638d8e",
        "status" : "skipped",
        "statusDetails" : "Skipped: missing notification",
        "time" : {
          "start" : 1650553594405,
          "stop" : 1650553594405,
          "duration" : 0
        }
      }, {
        "uid" : "9125d7c654cc7c4f",
        "reportUrl" : "https://ArkadiyVoronov.github.io/Test_with_a_team/4//#testresult/9125d7c654cc7c4f",
        "status" : "skipped",
        "statusDetails" : "Skipped: missing notification",
        "time" : {
          "start" : 1650552892728,
          "stop" : 1650552892728,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ "@pytest.mark.skip(reason='missing notification')" ]
  },
  "source" : "70968d80fde21448.json",
  "parameterValues" : [ ]
}